### DISCUSSION TITLE
Changing the license

### DISCUSSION THREAD

**Discussion by driedpampas** [cotes2020/jekyll-theme-chirpy#1204] [Q&A] [answered]
At the end of every post this is written: "This post is licensed under[ CC BY 4.0 ](https://creativecommons.org/licenses/by/4.0/)by the author.". How can I change this if I wanted to use another license? This is also present at the bottom of the page: "© 2023 [author](twitter.com/insert-autho-name). Some rights reserved." when hovering over "Some rights reserved.". Is this changeable if I'm using Chirpy Starter?

**Comment by cotes2020** [comment, marked as answer]
Yes, everything you mentioned can be changed.

Copy the `_data/locales/{lang}.yml` file to the same path in your project and then make changes in that YAML file to override the default settings.

**Reply by driedpampas** [reply]
Thanks, that helped but how do I also change this
![](https://i.imgur.com/a70poDx.png)

**Reply by cotes2020** [reply]
Ummm
https://github.com/cotes2020/jekyll-theme-chirpy/blob/62bcd601fcadc602c81672b1d4b937231396c3c0/_data/locales/en.yml#L37-L41

**Reply by driedpampas** [reply]
Oh, so I'm blinder than I thought. Thanks for the help.

**Reply by unbreakabl3** [reply]
Hey, @cotes2020 .
I did precisely the same steps, and nothing changed. The license footers are still there. Anything I am doing wrong?

**Reply by jackusay** [reply]
![2024-08-18 14_07_56-jekyll-theme-chirpy__data_locales_en yml at master · cotes2020_jekyll-theme-chir](https://github.com/user-attachments/assets/39c9e84a-7f1e-4184-89fa-cb821f88bc2a)

change below lines:
template: This post is licensed under :LICENSE_NAME by the author.
brief: Some rights reserved.
